xfetus/xfetus.py

```diff
@@ -77,12 +91,17 @@ def __getitem__(self, idx):
 
         ## Downsample image for SRGAN
         ds_image = resize(
-            image.cpu().numpy(),
+            image,
```

Comment:
On my setup I need to use `image.cpu().numpy()` to solve `TypeError: Cannot interpret 'torch.float32' as a data type`. Let's double that we have the right data format.